src/dp_abstraction/aca_dataplane_ovs.cpp

```diff
@@ -911,6 +919,14 @@ int ACA_Dataplane_OVS::update_neighbor_state_workitem(NeighborState current_Neig
   auto operation_total_time =
           cast_to_microseconds(operation_end - operation_start).count();
 
+  auto validate_info_total_time =
+          cast_to_microseconds(found_subnet_info_time - fixed_ip_loop_start).count();
+
+  auto determine_same_host_total_time =
+          cast_to_microseconds(determined_same_host_time - found_subnet_info_time)
+                  .count();
+  auto update_neighbor_total_time =
+          cast_to_microseconds(update_neighbor_time - determined_same_host_time);
```

Comment:
need _.count()_ like other lines?

Reply:
Good catch. @tianyuan129 , please add the `.count()` at the end of line 929, just like line 927.

Reply:
Fixed